src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py

```diff
@@ -853,7 +853,7 @@
 
 helps['snapshot'] = """
     type: group
-    short-summary: Manage snapshots of Virtual Machines.
+    short-summary: Manage point-in-time copies of a managed disk, native blobs, or another snapshots.
```

Comment:
Recommended tweak: Manage point-in-time copies of managed disks, native blobs, or other snapshots.